zcncore/wallet.go

```diff
@@ -627,6 +632,24 @@ func GetBalance(cb GetBalanceCallback) error {
 	return nil
 }
 
+// GetBalance retreives wallet nonce from sharders
+func GetNonce(cb GetNonceCallback) error {
+	err := checkConfig()
+	if err != nil {
+		return err
+	}
+	go func() {
+		value, info, err := getNonceFromSharders(_config.wallet.ClientID)
+		if err != nil {
+			Logger.Error(err)
+			cb.OnNonceAvailable(StatusError, 0, info)
```

Comment:
Check if `cb` is nil to avoid potential panic perhaps.

Reply:
fixed